runtime_builders/builder_util.py

```diff
@@ -69,8 +144,19 @@ def load_manifest_file():
         command = ['gsutil', 'cp', MANIFEST_FILE, tmp]
```

Comment:
"gsutil cat" might be a bit easier to use here, you could avoid writing a tmp file.